app/controllers/concerns/decidim/consultations/needs_question.rb

```diff
@@ -10,17 +10,19 @@ def self.enhance_controller(instance_or_module)
           helper_method :current_question, :current_consultation, :stats
 
           helper Decidim::WidgetUrlsHelper
+          helper Decidim::ActionAuthorizationHelper
         end
       end
 
       def self.extended(base)
         base.extend Decidim::NeedsOrganization, InstanceMethods
+        base.include Decidim::Consultations::ActionAuthorization
```

Comment:
Shouldn't this be `base.extend`?

Reply:
ActionAuthorization implements:

```ruby
extend ActiveSupport::Concern
included do
   helper_method :action_authorization
end
```
AFAIK it only does its magic when its included.